src/internal.c

```diff
@@ -26693,13 +26693,16 @@ const char* wolfSSL_get_cipher_name_iana(WOLFSSL* ssl)
 }
 
 int GetCipherSuiteFromName(const char* name, byte* cipherSuite0,
-                           byte* cipherSuite, int* flags)
+                       byte* cipherSuite, byte* major, byte* minor, int* flags)
```

Comment:
why is this important for CID?

Reply:
This was important for the test. `wolfSSL_set_cipher_list` was not limiting the ciphers correctly because not all the ciphers were recognized as TLS 1.3